scripts/install-forms-flow.sh

```diff
@@ -1,47 +1,115 @@
 #!/bin/bash
-read -p "Install using forms-flow package registry? (y/n):" IS_FROM_REGISTRY
-read -p "Please enter the domain name (ex: apps.bronze.aot-technologies.com):" DOMAIN_NAME
-read -p "Please enter the namespace (ex: forms-flow):" NAMESPACE
-read -p "Is this a premium installation? (y/n):" IS_PREMIUM
 
-echo "Installing forms flow ..."
+# Exit when script tries to use undeclared variables
+set -o nounset
 
-DIRECTORY="../charts"
+# Assign versions
+# See https://github.com/AOT-Technologies/forms-flow-ai-charts/releases
+version_ff_admin="v2.1.1"
```

Comment:
This approach would need change to this file whenever a new version is published on any component. can we always use the latest by default ? Or ask user for specific version.